source/state_representation/include/state_representation/trajectory/CartesianTrajectory.hpp

```diff
@@ -92,6 +92,19 @@ class CartesianTrajectory : public TrajectoryBase<CartesianTrajectoryPoint> {
    */
   void set_reference_frame(const CartesianPose& pose);
 
+  /**
```

Comment:
Since you defined the getters at the top in the base class, I like to have the same order in the derived classes :stuck_out_tongue_closed_eyes: